app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,63 @@
+$font-color: #088da5;
+$black: #000000;
+
+body {
+
+  background-color: $black;
+
+}
+
+h1,
+h3 {
+
```

Comment:
Sorry to be Hound-ish, but get rid of the blank lines inside your curly braces here! (I guess unless Hound explicitly told you to add those in... but that would be weird)